src/Layers/FeatureLayer/FeatureLayer.js

```diff
@@ -90,8 +99,23 @@ EsriLeaflet.Layers.FeatureLayer = EsriLeaflet.Layers.FeatureManager.extend({
         return;
       }
 
+      // polylines and polygons
       if (layer && layer.setLatLngs) {
-        this._updateLayerGeometry(layer, geojson);
+        this._updateLayer(layer, geojson);
+        // update custom symbology for layer if necessary
+        this.resetStyle(geojson.id);
+        return;
+      }
+
+      // points
+      if (layer && layer.setLatLng) {
```

Comment:
nevermind.  i see what you're saying now.  i'll rewrite this after i confirm whats needed.